test/delay-test.js

```diff
@@ -53,6 +54,16 @@ describe('Delayed jobs', (it) => {
     });
 
     function makeQueue(...args) {
+      if (redisUrl) {
+        if (args.length === 0) {
+          args.push({});
+        }
+        if (!args[0].redis) {
```

Comment:
```suggestion
        } else if (!args[0].redis) {
```

Reply:
I've made this change in my rebased branch.

Reply:
Actually, I'm not sure this is a correct fix.  Even if no args are provided, we want to use the redisUrl.  And, all uses of `makeQueue()` in this file provide arguments, so the no-arg case is never exercised!  I'm going to leave it as it was because that should provide the least surprise if/when `makeQueue()` is called with no arguments.